Code/Geometry/Wrap/Trajectory.cpp

```diff
@@ -0,0 +1,145 @@
+// $Id$
+//
+//  Copyright (C) 2016 Paolo Tosco
+//
+//   @@ All Rights Reserved @@
+//  This file is part of the RDKit.
+//  The contents are covered by the terms of the BSD license
+//  which is included in the file license.txt, found at the root
+//  of the RDKit source tree.
+//
+#include <RDBoost/python.h>
+
+#include <RDBoost/Wrap.h>
+#include <Geometry/Trajectory.h>
+
+namespace python = boost::python;
+
+namespace RDGeom {
+
+class PySnapshot {
```

Comment:
I'm a bit confused why this class needs to exist.  Could you indicate what the benefit is as opposed to just wrapping a Snapshot class?